packages/react/src/components/DetailsList/DetailsRow.base.tsx

```diff
@@ -363,7 +368,7 @@ export class DetailsRowBase extends React.Component<IDetailsRowBaseProps, IDetai
         <GroupSpacer
           indentWidth={indentWidth}
           role="gridcell"
-          count={groupNestingDepth! - (this.props.collapseAllVisibility === CollapseAllVisibility.hidden ? 1 : 0)}
+          count={totalNestingDepth! - (this.props.collapseAllVisibility === CollapseAllVisibility.hidden ? 1 : 0)}
```

Comment:
the alignment

Reply:
I did not change the RowFields columnStartIndex.  ( The default DetailsRowFields does not use it )
```
            <RowFields
              rowClassNames={this._rowClassNames}
              rowHeaderId={`${id}-header`}
              columns={[columnMeasureInfo.column]}
              item={item}
              itemIndex={itemIndex}
              columnStartIndex={(showCheckbox ? 1 : 0) + (groupNestingDepth ? 1 : 0) + columns.length} //***********
              onRenderItemColumn={onRenderItemColumn}
              getCellValueKey={getCellValueKey}
``